src/assets/thirdPartyServices/snet/zeta36_chess_alpha_zero/alpha_zero_pb.js

```diff
@@ -0,0 +1,440 @@
+/**
+ * @fileoverview
+ * @enhanceable
+ * @public
+ */
+// GENERATED CODE -- DO NOT EDIT!
+
+var jspb = require('google-protobuf');
+var goog = jspb;
+var global = Function('return this')();
+
+goog.exportSymbol('snet_zeta36_chess_alpha_zero.Input', null, global);
+goog.exportSymbol('snet_zeta36_chess_alpha_zero.Output', null, global);
+
```

Comment:
We don't need `eslint-disable` here?

Reply:
fixed in [commit](https://github.com/singnet/refactored-dapp/pull/122/commits/9a3e85577d17f47d863793d9a9207f15d36af266)